lightning-block-sync/src/convert.rs

```diff
@@ -156,6 +156,14 @@ impl TryInto<(BlockHash, Option<u32>)> for JsonResponse {
 	}
 }
 
+impl TryInto<serde_json::Value> for JsonResponse {
```

Comment:
Why TryInto instead of just making the inner field pub?

Reply:
No reason. Is that better?

Reply:
Just seems simpler, plus it avoids an unwrap that cannot fail at the callsite.

Reply:
`Into` is the non-failing version, but I agree making the inner field `pub` would be simpler.